src/tmx/TmxUtils/src/simulation/SimulationEnvUtils.h

```diff
@@ -51,6 +51,11 @@ namespace tmx::utils::sim{
          * for CDASim connection.
          */
         constexpr inline static const char *LOCAL_IP = "LOCAL_IP";
+        /**
+         * @brief Name of environment variable for storing infrastructure id of V2xhub. Only necessary in SIMULATION MODE 
+         * for CDASim connection.
+         */
+        constexpr inline static const char *INFRASTRUCTURE_ID = "INFRASTRUCTURE_ID";
```

Comment:
Please update the README.md Simulation section with this new environment variable.

Reply:
added